index.html

```diff
@@ -3,8 +3,10 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic|Material+Icons">
     <link rel="shortcut icon" type="image/ico" href="static/favicon.jpg" />
+    <link href="https://afeld.github.io/emoji-css/emoji.css" rel="stylesheet">
```

Comment:
What's the usage of this css?

Reply:
I thought that we have to make it stick to the bottom of the page so that it doesn't overlap the the pokemon template (and the css was by mistake).
I changed it now. Do have a look now, please.